build-npm-module/index330.js

```diff
@@ -0,0 +1,10 @@
+const caseConverter = require('./index');
+
+const str = 'hello free Code Camp!';
+console.log(caseConverter.getUpperCase(str));
+console.log(caseConverter.getLowerCase(str));
+console.log(caseConverter.getProperCase(str));
+
+// Let's try your getSentenceCase function. Conver `str` to sentence case and print it.
```

Comment:
Finally, let's try your `getSentenceCase()` function. Convert `str` to sentence case and print it to the console.